examples/llm/main.py

```diff
@@ -39,6 +39,7 @@ def main(cfg):
     )
 
     reproducibility.seed_all(cfg.seed)
+    dist.initialize_dist(get_device(None))
```

Comment:
can we not do
```python
    dist.initialize_dist(get_device())
```
or just 
```
dist.initialize_dist()
```
? I assume not since you wrote it this way; if not, we should open a Composer usability issue

Reply:
I will open a Composer usability issue